docs/readme.md

```diff
@@ -0,0 +1,43 @@
+# UDMI specification and tools
+
+## UDMI specification
+
+* UDMI definition
+* [Technology stack](tech_stack.md)
+* Message types
+  * metadata.json
+  * envelope.json
+  * state.json
+  * event.json
+  * config.json
+  * discover.json
+* UDMI site model
+* UDMI transactions
```

Comment:
nit: I think we should be consistent and use "sequences" -- (e.g. there is a bin/sequencer already for testing things)